src/ui/map.js

```diff
@@ -1428,6 +1428,9 @@ class Map extends Camera {
         const container = this._container;
         container.classList.add('mapboxgl-map');
 
+        const missingCSSContainer = DOM.create('div', 'mapboxgl-missing-css', container);
```

Comment:
This div should either be removed in `remove()`, or go inside the `mapboxgl-control-container` div (which is already removed in `remove()`).

Reply:
Thanks for the review! I've updated the PR.